src/main/scala/org/broadinstitute/hail/expr/AST.scala

```diff
@@ -308,7 +308,9 @@ case class Select(posn: Position, lhs: AST, rhs: String) extends AST(posn, lhs)
       AST.evalCompose[IndexedSeq[_]](ec, lhs)(_.tail)
 
     case (t, name) => FunctionRegistry.lookupField(ec)(t, name)(lhs)
-      .getOrElse(fatal(s"No such method `$name' on object of type `$t'"))
+      .getOrElse(fatal(s"""`$t' has no field `$name'
```

Comment:
We don't currently have a distinction between nullary *methods* and fields. I've been calling functions anything that isn't invoked with a dot.